.husky/pre-commit

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-yarn lint-staged
```

Comment:
Are we pushing this config to `package.json`?

Reply:
Reverted.